flink-java/src/main/java/org/apache/flink/api/java/sampling/PoissonSampler.java

```diff
@@ -28,11 +29,16 @@
  *
  * @param <T> The type of sample.
  * @see <a href="https://en.wikipedia.org/wiki/Poisson_distribution">https://en.wikipedia.org/wiki/Poisson_distribution</a>
+ * @see <a href="http://erikerlandson.github.io/blog/2014/09/11/faster-random-samples-with-gap-sampling/">Gap Sampling</a>
  */
 public class PoissonSampler<T> extends RandomSampler<T> {
 	
 	private PoissonDistribution poissonDistribution;
 	private final double fraction;
+	private final Random random = new Random();
```

Comment:
The "random" instance should better to be initialized in constructor method and use the seed parameter while available.